spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala

```diff
@@ -257,6 +312,46 @@ class Graph[T: ClassTag](val inputs : Seq[ModuleNode[T]],
     }
   }
 
+
+  /**
+   * set an array of layers to be trainable, the rest of layers are freezed
```

Comment:
why are the rest layers frozen? Why you change the state of other layers? The method should only care what passed in. It should NEVER touch the others. DO NOT design API like this.